doc/mgmt/SONiC_OC_FDB_get_support_HLD.md

```diff
@@ -0,0 +1,271 @@
+
+Syntax Description:
+
+|    Keyword    | Description |
+|:-----------------:|:-----------:|
+| address `mac` | This option dumps the MACs matching the particular Mac
+| count | This option provides the no. of MAC entries present in the system
+| static | This option dumps the MAC entries matching the static entry type
+| dynamic | This option dumps the MAC entries matching the dynamic entry type
+| interface | This option dumps the MACs matching the particular physical/port-channel interface
+| vlan | This option dumps the MACs matching the particular VLAN
+
+
+Command Mode: User EXEC
+
+Example:
+````
+sonic# show mac address-table
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+20      00:11:22:33:44:55       static          Ethernet0
+sonic#
+
+sonic# show mac address-table count
+MAC Entries for all vlans :
+Dynamic Address Count :                  1
+Static Address (User-defined) Count :    1
+Total MAC Addresses in Use:              2
+sonic#
+
+
+sonic# show mac address-table address 00:11:22:33:44:55
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table dynamic
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+sonic#
+
+sonic# show mac address-table dynamic address 90:b1:1c:f4:9d:83
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+
+sonic# show mac address-table vlan 1
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+
+sonic# show mac address-table interface Ethernet 1
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+sonic#
+
+sonic# show mac address-table interface Ethernet 1
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+sonic#
+
+sonic# show mac address-table static
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table static address 00:11:22:33:44:55
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table static vlan 20
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+
+sonic# show mac address-table static interface Ethernet 1
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table vlan 20
```

Comment:
This one is a repeat of a previous example

Reply:
done